Shared/Prefs.swift

```diff
@@ -14,7 +14,7 @@ public struct PrefsKeys {
     public static let KeyMailToOption = "MailToOption"
     public static let HasFocusInstalled = "HasFocusInstalled"
     public static let HasPocketInstalled = "HasPocketInstalled"
-    public static let IntroSeen = "IntroViewControllerSeen"
+    public static let IntroSeen = "IntroViewControllerSeen-2"
```

Comment:
how about adding a version string here instead? 

```suggestion
    public static let IntroSeen = "IntroViewControllerSeen-3.0.0"
```